test/GetEventStore.Tests/Defaults.cs

```diff
@@ -0,0 +1,44 @@
+﻿using EventSourcing.Tests;
+using Orleans.Storage;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TestExtensions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace GetEventStore.Tests
+{
+    [Collection(TestEnvironmentFixture.DefaultCollection)]
+    [TestCategory("Functional")]
+    public class Defaults : EventStoreTests, IClassFixture<ProvidersFixture>
```

Comment:
These are the main tests, right? I would name it something like GetEventStoreProviderTests.
It took me a long time to understand where the important tests were.